lemur/plugins/lemur_entrust/plugin.py

```diff
@@ -78,11 +78,35 @@ def process_options(options):
         "eku": "SERVER_AND_CLIENT_AUTH",
         "certType": product_type,
         "certExpiryDate": validity_end,
+        # "keyType": "RSA", Entrust complaining about this parameter
```

Comment:
the `client_id` is missing from the data.

Reply:
*sigh* Had problems syncing my dev-enviroment. So somehow the two lines got lost. Thank you. I repaired it.